<launch>

    <!-- Turtlesim Node-->
    <node pkg="pva_tracker" type="square_fly" name="square_fly" />

    <node pkg="pva_tracker" type="mpc_tracker_pytorch" name="mpc_tracker_pytorch" output="screen"/>

    <node pkg="pva_tracker" type="tracker_nn.py" name="tracker_nn" output="screen"/>
    <!--node pkg="rosbag" type="record" name="test_take_off_record" args="record -O /home/pengpeng/catkin_ws/src/pva_tracker/bag/hover_test_on_up_record_$(date +%H%M%S) /mavros/local_position/pose /test_take_off "/-->



</launch>
